docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.8'
+
+services:
+
+  mysql_server:
+    image: mysql:latest
+    restart: always
+    environment:
```

Comment:
sercrety chyba powinny leciec z enva, zobacz czy da sie przypisac zmienne srodowiskowe tutaj
https://stackoverflow.com/questions/29377853/how-to-use-environment-variables-in-docker-compose